usegalaxy.org/text_manipulation.yml.lock

```diff
@@ -93,3 +93,8 @@ tools:
   - 60ff16842fcd
   tool_panel_section_id: text_manipulation
   tool_panel_section_label: Text Manipulation
+- name: regex_find_replace
+  owner: galaxyp
+  revisions: []
```

Comment:
```suggestion
  revisions:
  - ae8c4b2488e7
```
you need to run `TOOLSET=usegalaxy.org make fix` to get the revisions

Reply:
Yep, it appears I did incorrectly. Thanks!